test.clj

```diff
@@ -1,7 +1,38 @@
 ;;; font locking
 (ns clojure-mode.demo
-  (:require [clojure.something]
-            [something.s]))
+  (:require
+   [oneword]
+   [seg.mnt]
+   [mxdCase]
+   [CmlCase]
+   [veryCom|pLex.stu-ff]))
+
+;; TODO the face of 'fn' is 't'. Where is it defined?
+(fn foo [x] x)
```

Comment:
`fn` is in the list of special forms, so it should be font-locked as one. I can't imagine what can cause the problem you've mentioned. At least I don't see anything obvious in your changes.

Reply:
I need to have a look at it. Give me some time please.